pycircleci/api.py

```diff
@@ -747,6 +748,40 @@ def delete_envvar(self, username, project, name, vcs_type=GITHUB):
         resp = self._request(DELETE, endpoint)
         return resp
 
+    def add_context(self, context, username, vcs_type=GITHUB):
+        """Get contexts for an organization
+
+        :param context:  Context name to add.
+        :param username: Org or user name.
+        :param vcs_type: VCS type (github, bitbucket). Defaults to ``github``.
+
+        Endpoint:
+            POST ``/v2/context``
+        """
+
+        slug = self.owner_slug(username, vcs_type)
+        data = json.dumps({
```

Comment:
not sure this is needed. I think `requests` takes care of it. see other POST examples.